src/solvers/dg_unstructured_quad/dg.jl

```diff
@@ -0,0 +1,18 @@
+
+# 2D unstructured DG implementation
+include("dg_element_geometry_2d.jl")
+include("containers_2d.jl")
+include("dg_2d.jl")
+
+@inline ndofs(mesh::UnstructuredQuadMesh, dg::DG, cache) = nelements(cache.elements) * nnodes(dg)^ndims(mesh)
+
+
+@inline function get_one_sided_surface_node_vars(u, equations, solver::DG, j, indices...)
+  # There is a cut-off at `n == 10` inside of the method
+  # `ntuple(f::F, n::Integer) where F` in Base at ntuple.jl:17
+  # in Julia `v1.5`, leading to type instabilities if
+  # more than ten variables are used. That's why we use
+  # `Val(...)` below.
+  u_surface = SVector(ntuple(v -> u[j, v, indices...], Val(nvariables(equations))))
+  return u_surface
+end
```

Comment:
Isn't this just `get_node_vars`? IMHO, in this case you could (should?) just use `get_node_vars` instead of duplicating the function, but I am not 100% sure. What's your take on this @ranocha ?

Reply:
Now that I see how it is used in e.g., https://github.com/trixi-framework/Trixi.jl/pull/500/files#diff-a8b4ce5ea2ac497368f7401e00f762da3a8cd791c955c98571c62d2fc04ef428R186-R187, I am not sure anymore... maybe it is ok to have this extra implemenation :thinking:

Reply:
Well, it's acting on `interfaces.u`, so it should probably not be the same as `get_node_vars`, which acts on all states `u`.

Reply:
Yes, yes, this was born out of my lack of focus today... closing this